api/substrate.config.yaml

```diff
@@ -1,3 +1,8 @@
+### Define the service
+service_name:
```

Comment:
Not 100% on this, but maybe this will be more clear if it's written as `[service_name]` to show that it's intended to be variable.